controllers/contact-controller.js

```diff
@@ -160,6 +184,17 @@ function buildSearchQueryObject(body) {
   };
 }
 
+function isNonDataCallType(callType) {
+  return callType !== callTypes.caller && callType !== callTypes.child;
+}
```

Comment:
Same question as above, I'm copy pasting for now, but I fell like this shouldn't be here